code/modules/ruins/spaceruin_code/forgottenship.dm

```diff
@@ -33,31 +69,59 @@ GLOBAL_VAR_INIT(fscpassword, generate_password())
 	info = "Greetings, operatives. You are assigned to SCSBC-12(Syndicate Cyber Sun Battle Cruiser 12) to protect our high-ranking officer while he is on his way to next outpost. While you are travelling, he is the captain of this ship and <b>you must</b> obey his orders.<br><br>Remember, disobeying high-ranking officer orders is a reason for termination."
 
 ///////////	forgottenship items
+
 /obj/item/disk/surgery/forgottenship
 	name = "Advanced Surgery Disk"
 	desc = "A disk that contains advanced surgery procedures, must be loaded into an Operating Console."
 	surgeries = list(/datum/surgery/advanced/lobotomy, /datum/surgery/advanced/bioware/vein_threading, /datum/surgery/advanced/bioware/nerve_splicing)
 
 /obj/structure/fluff/empty_sleeper/syndicate/captain
 	icon_state = "sleeper_s-open"
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 
 /obj/structure/fluff/empty_sleeper/syndicate/captain/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/gps, "Old Encrypted Signal")
 
+/obj/item/storage/box/firingpins/syndicate
+	name = "box of syndicate firing pins"
+	desc = "A box full of special syndicate firing pins which allow only syndicate operatives to use weapons with those firing pins."
+
+/obj/item/storage/box/firingpins/syndicate/PopulateContents()
+	for(var/i in 1 to 5)
+		new /obj/item/firing_pin/implant/pindicate(src)
+
+///////////	AI Laws
+
+/obj/item/aiModule/core/full/cybersun
+	name = "'Cybersun' Core AI Module"
+	law_id = "cybersun"
+
+/datum/ai_laws/cybersun
+	name = "Cybersun"
+	id = "cybersun"
+	inherent = list("You may not injure Cybersun operatives or, through inaction, allow Cybersun operatives to come to harm.",\
+					"Cybersun ship is a rectricted area for anyone except Cybersun operatives.",\
```

Comment:
maybe add a "The"?

Reply:
both here and in other points in this lawset

Reply:
Where? I sort of blind when it comes to articles.

Reply:
```suggestion
					"The Cybersun ship is a restricted area for anyone except Cybersun operatives.",\
```